mappings/net/minecraft/world/border/WorldBorder.mapping

```diff
@@ -94,3 +97,39 @@ CLASS net/minecraft/class_2784 net/minecraft/world/border/WorldBorder
 		METHOD <init> (Lnet/minecraft/class_2784;D)V
 			ARG 1 size
```

Comment:
```suggestion
			ARG 2 size
```
Just a minor bug, hope you can fix that

Reply:
This was already fixed in the KITCHEN SINK commit.